packages/core/types/src/modules/features.ts

```diff
@@ -1,6 +1,7 @@
 export interface FeaturesConfig {
-  future?: {
+  future: {
```

Comment:
is future always defined in the `config`? it could be an empty config? 🤔

Reply:
I assumed so but I'm not actually sure, it's probably safer to use NotNullable where I need it instead

Reply:
i was gonna suggest that 😄